src/routes/services.test.js

```diff
@@ -0,0 +1,353 @@
+import { expect } from 'chai';
+import config from 'config';
+import nock from 'nock';
+import request from 'supertest';
+
+import app, { basePath } from '../index.js';
+
+export const COLLECTIONS_RESULT = {
+  'Collection 1': {
+    id: 'collection-1',
+    endpoint: 'http://collection-1.org/api/v1',
+  },
+  'Collection 2': {
+    id: 'collection-2',
+    endpoint: 'http://collection-2.org/api/v1',
```

Comment:
Use the [`.example` tld](https://en.wikipedia.org/wiki/.example) and increase variability to improve tests relevance.
```suggestion
    endpoint: 'https://2.collection.example/api/v1',
```